### DISCUSSION TITLE
Cuda out of memory

### DISCUSSION THREAD

**Discussion by 3majsie** [victorca25/traiNNer#68] [Q&A] [answered]
I make the SR 1x model for removing water models, of course only for learning :) What to do not to turn off the computer after finishing training the model each time, to reset the memory "Cuda out of memory". I can start training for the night, by the morning the model was ready, is it possible?
![Cuda](https://user-images.githubusercontent.com/107211998/172925019-d2cc0943-8010-4941-bb07-5cf5850ee0a1.png)

**Comment by victorca25** [comment, marked as answer]
Hello! Typically 1x models will consume more VRAM than 4x models if the same configuration is used (specially _batch size_ and _crop size_).

If using the standard ESRGAN architecture, the recommendation here is to reduce either the batch size, the crop size or both, so the tensors fit in memory.

Alternatively, you can try to use the [pixel-unshuffle wrapper](https://github.com/victorca25/traiNNer/blob/master/docs/unshuffle.md#smaller-downscale-factor), but that depends on how you plan to use the model after it's trained and if it that code supports using the models with the wrappers.

**Comment by 3majsie** [comment]
Thanks to Victoria also gives her Train_sr.yml maybe you will come up with an idea what can be improved. Thanks again!
* sorry for my english, i come from poland and i use google translate ;)
[train_sr.txt](https://github.com/victorca25/traiNNer/files/8881013/train_sr.txt)
